compiler/rustc_builtin_macros/src/assert.rs

```diff
@@ -85,7 +85,7 @@ pub fn expand_assert<'cx>(
                 DUMMY_SP,
                 Symbol::intern(&format!(
                     "assertion failed: {}",
-                    pprust::expr_to_string(&cond_expr).escape_debug()
+                    pprust::expr_to_string(&cond_expr)
```

Comment:
we could, perhaps, just `.replace("\n", "")` on this str?

Reply:
Why? To make it fit on a single line? But a multi-line expression with have one or more indents in it, and they'll show up in the middle of the one line string.

Reply:
Ah, yeah, I guess we would have to prune those idents too. Nvm 😸